src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/test-crash-reporting.mdx

```diff
@@ -0,0 +1,472 @@
+---
+title: Test crash reporting
+type: apiDoc
+shortDescription: 'Throws a demo run-time exception to test New Relic crash reporting.'
+tags:
+  - Mobile monitoring
+  - Mobile SDK API
+  - Custom instrumentation
+metaDescription: Mobile app monitoring API used to throw a demo run-time exception to test New Relic crash reporting.
+redirects:
+- /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/crashnow-ios-sdk-api
+- /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/crashnow-android-sdk-api
+---
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="android">
+            Android
+        </TabsBarItem>
+        <TabsBarItem id="ios">
+            iOS
+        </TabsBarItem>
+        <TabsBarItem id="capacitor">
+            Capacitor
+        </TabsBarItem>
+        <TabsBarItem id="cordova">
+            Cordova
+        </TabsBarItem>
+        <TabsBarItem id="flutter">
+            Flutter
+        </TabsBarItem>
+        <TabsBarItem id="react">
+            React Native
+        </TabsBarItem>
+        <TabsBarItem id="xamarin">
+            Xamarin
+        </TabsBarItem>
+        <TabsBarItem id="maui">
+            .NET MAUI
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="android">
+## Syntax [#syntax]
+
+### Java [#java]
+
+```java
+NewRelic.crashNow(string $message)
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+NewRelic.crashNow(message:String!)
+```
+
+## Description [#description]
+
+Throws a demo run-time exception named java.lang.RuntimeException to test New Relic crash reporting.
+
+With this method, you can throw a demo run-time exception named `java.lang.RuntimeException` with a message. This crash will appear in the [**Crash Analysis** page](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/crash-analysis-group-filter-your-crashes).
+
+## Parameters [#parameters]
+
+<table>
+  <thead>
+    <tr>
+      <th width={200}>
+        Parameter
+      </th>
+      <th width={200}>
+        Type
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `$message`
+      </td>
+
+            <td>
+        `string`
+      </td>
+
+      <td>
+        Optional. A message attached to the exception.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Example [#example]
+
+### Java [#java]
+```java
+NewRelic.crashNow("This is my test message");
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
```

Comment:
@ndesai-newrelic Can you please provide a kotlin example equivalent to the Java example?

Reply:
NewRelic.crashNow()
 NewRelic.crashNow("test crash")